router.go

```diff
@@ -532,7 +581,10 @@ func New(docs, version string) *Router {
 		return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 			// Inject the operation info before other middleware so that the later
 			// middleware will have access to it.
-			req = req.WithContext(context.WithValue(req.Context(), opIDContextKey, &OperationInfo{}))
+			reqContext := req.Context()
+			withOpenID := context.WithValue(reqContext, opIDContextKey, &OperationInfo{})
+			withRouter := context.WithValue(withOpenID, routerContextKey, r)
+			req = req.WithContext(withRouter)
```

Comment:
```suggestion
			reqContext := req.Context()
			withOpID := context.WithValue(reqContext, opIDContextKey, &OperationInfo{})
			withRouter := context.WithValue(withOpID, routerContextKey, r)
			req = req.WithContext(withRouter)
```

Reply:
This is fine for me.  Note that I was just following the naming convention of the context key, `opIDContextKey`.  So this change is fine but I'd say that for consistency you should then rename `opIDContextKey` to `openIDContextKey`.